index.ts

```diff
@@ -92,13 +92,39 @@ async function main() {
             }
             continue;
           }
+
+          // if the project issue is assigned to someone and it's open in the devpool, then close it
+          if (projectIssue.assignee?.login && devpoolIssue.state === "open") {
+            await octokit.rest.issues.update({
+              owner: DEVPOOL_OWNER_NAME,
+              repo: DEVPOOL_REPO_NAME,
+              issue_number: devpoolIssue.number,
+              state: "closed",
+            });
+            console.log(`Closed (assigned): ${devpoolIssue.html_url} (${projectIssue.html_url})`);
+            continue;
+          }
+
+          // if the project issue is not assigned to someone and it's closed in the devpool, then reopen it
+          if (!projectIssue.assignee?.login && devpoolIssue.state === "closed") {
```

Comment:
If project issue is closed as "not planned" without an assignee then such issue will be reopened in the devpool directory while the expected behaviour is to keep it closed.

I guess here we should also check that `projectIssue.state === "open"`

Example:
Closed project issue without assignee: https://github.com/rndquu-org/test-repo/issues/51
Reopened issue: https://github.com/rndquu/devpool-directory/issues/64